lunes_cms/static/js/toggle_document_form_fields.js

```diff
@@ -12,7 +12,7 @@ $(document).ready(() => {
   $("#id_word_type").trigger("change");
 });
 
-//Toggle the drop down for grammatical gender depending on the chosen word type
+// Toggle the drop down for grammatical gender depending on the chosen word type
 $(document).ready(() => {
```

Comment:
`$(document).ready(() => {
  const toggleGenderDropdown = () => {
    const isNoun = $("#id_word_type").val() === "Nomen";
    $("#id_grammatical_gender").closest("div.row").toggle(isNoun);
  };

  $("#id_word_type").change(toggleGenderDropdown);
  toggleGenderDropdown();
});`

I think this is a bit better to read?